client/src/Pages/Dashboard/DashboardView.js

```diff
@@ -1,90 +1,90 @@
-import React from "react";
-import { useQuery } from "@apollo/react-hooks";
-import {
-  getDashboardAllDataQuery,
-} from "../../queries/queries";
+import React, { useState } from 'react'
+import { useQuery } from '@apollo/react-hooks'
+import { getDashboardAllDataQuery } from '../../queries/queries'
+import LoadingSpinner from '../../Components/LoadingSpinner'
+import CarList from '../../Components/CarList'
+import DashboardHeader from '../../Components/DashboardHeader'
+import { Card } from 'semantic-ui-react'
+import LastFillUpCard from '../../Components/Card/LastFillUpCard'
+import FuelLeftCard from '../../Components/Card/FuelLeftCard'
+import DiagnosticCard from '../../Components/Card/DiagnosticCard'
+import BusinessRatioCard from '../../Components/Card/BusinessRatioCard'
+import AverageSpeedCard from '../../Components/Card/AverageSpeedCard'
+import TravelDistanceTotalCard from '../../Components/Card/TravelDistanceTotalCard'
+import TimeInCarCard from '../../Components/Card/TimeInCarCard'
+import EmissionsCard from '../../Components/Card/EmissionsCard'
+import FuelEconomyCard from '../../Components/Card/FuelEconomyCard'
+import GoogleMap from '../../Components/GoogleMap/GoogleMap'
+import AntdTableComponent from '../../Components/AntdTable/AntdTableComponent'
+import Footer from '../../Components/Footer'
 
-import LoadingSpinner from "../../Components/LoadingSpinner";
-import CarList from "../../Components/CarList";
-import DashboardHeader from "../../Components/DashboardHeader";
-import { Card } from "semantic-ui-react";
-import LastFillUpCard from "../../Components/Card/LastFillUpCard";
-import FuelLeftCard from "../../Components/Card/FuelLeftCard";
-import DiagnosticCard from "../../Components/Card/DiagnosticCard";
-import BusinessRatioCard from "../../Components/Card/BusinessRatioCard";
-import AverageSpeedCard from "../../Components/Card/AverageSpeedCard";
-import TravelDistanceTotalCard from "../../Components/Card/TravelDistanceTotalCard";
-import TimeInCarCard from "../../Components/Card/TimeInCarCard";
-import EmissionsCard from "../../Components/Card/EmissionsCard";
-import FuelEconomyCard from "../../Components/Card/FuelEconomyCard";
-
-import GoogleMap from "../../Components/GoogleMap/GoogleMap";
-import AntdTableComponent from "../../Components/AntdTable/AntdTableComponent";
-import Footer from "../../Components/Footer";
-
-export default function DashboardView(props) {
-  const { loading, error, data } = useQuery(getDashboardAllDataQuery, {
-    variables: {
-      vehicleId: "71427823-ebeb-4836-945d-802adc14779d"
+export default function DashboardView() {
+    const [dropdownSelection, setDropdownSelection] = useState(null)
+    const handleChange = dropdownSelection => {
```

Comment:
I left a comment earlier about the variable names being too generic. Can you scope them down so it's clear what is being changed?